addons/html_builder/static/src/core/grid_layout/grid_layout_plugin.js

```diff
@@ -214,7 +214,8 @@ export class GridLayoutPlugin extends Plugin {
             if (allowGridMode) {
                 // Toggle the grid mode if it is not already on.
                 if (!isRowInGridMode) {
-                    toggleGridMode(containerEl);
+                    const preserveSelection = this.dependencies.selection.preserveSelection;
+                    toggleGridMode(containerEl, preserveSelection);
```

Comment:
Why do you need to use preserveSelection in this case ?

Reply:
Normally I don't need it, but I kept it just in case, and it was either give it to the function or change the function to consider that it may not always be there and it was easier to just give it (it was imported before so always there, but now that it is a resource I need to pass it) :slightly_smiling_face: